### PR TITLE
Frontend Homework 7 complete

### PR DESCRIPTION
**Pull request by asish918**
## Drift Video
[Drift Video](https://video.drift.com/v/abIpPjB7sN0uoQW9CnfQTGFNGg0l3C6XVKPKLIgyySX4)

## Screenshots - 
Screenshot 2024-02-14 at 23 31 04

Screenshot 2024-02-14 at 23 31 11

Screenshot 2024-02-14 at 23 31 43

### CODE REVIEW

**Comment by asish918** [comment]
| Metric | Value |
| ------ | ----- |
| alert_status | <b>:white_check_mark: OK</b> |
| bugs | 1 |
| code_smells | 0 |
| reliability_rating | C (at least 1 Major Bug)<br/>Note: A being best, E being worst |
| security_rating | A (0 Vulnerabilities)<br/>Note: A being best, E being worst |
| vulnerabilities | 0 |

**Comment by Nitesh2503** [comment]
The commit message must be proper

**Comment by Nitesh2503** [comment]
Nice work on the UI and the interface

File: frontend/homework/7/server.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        socket.emit('message', formatMessage(botName, 'Welcome to KDU Chat App!!'));
+
+        socket.broadcast.to(user.room).emit('message', formatMessage(botName, `${user.username} has joined the chat`));
+
+        io.to(user.room).emit('roomUsers', {
+            room: user.room,
+            users: getRoomUsers(user.room)
+        })
+    });
+
```
Comment (Nitesh2503) [inline frontend/homework/7/server.js:35]: Format the properly

File: frontend/homework/7/server.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+const express = require('express');
```
Comment (Nitesh2503) [inline frontend/homework/7/server.js:1]: Store the messages/ error messages in the constants

**Pull request closed without merging**